src/h.js

```diff
@@ -1,22 +1,29 @@
-export function h(tag, data) {
-  var node
-  var stack = []
-  var children = []
+function addChild(children, val) {
+  if (val == null || typeof val === 'boolean') return
 
-  for (var i = arguments.length; i-- > 2; ) {
-    stack[stack.length] = arguments[i]
+  if (Array.isArray(val)) {
+    for (var j = 0; j < val.length; j++) {
```

Comment:
Hey, array caching has since changed. I've tested this extensively, for early `let i=0` too. It's more of a  V8 "bug" right now, but has been present for about a year & does not look to be changing soon.